src/api/app/controllers/tag_controller.rb

```diff
@@ -146,11 +144,9 @@ def get_tags_by_user_and_package( do_render = true )
     @project = @package.project
 
     @tags = @package.tags.where("taggings.user_id = ?", user.id).order(:name)
-    if do_render
-      render partial: "tags"
-    else
-      return @tags
-    end
+    return @tags unless do_render
+
```

Comment:
According to [Ruby style guide](https://github.com/bbatsov/ruby-style-guide#empty-lines-between-methods):

> Use empty lines between method definitions and also to break up methods into logical paragraphs internally.

And this is not only something for Ruby, as it is something standard in most programming languages to use blank lines inside methods only to separate logical sections, for example you can find it in the [Python](https://www.python.org/dev/peps/pep-0008/#blank-lines) and [Java](http://www.oracle.com/technetwork/java/javase/documentation/codeconventions-141388.html) style guides.

And I would say than in most of the cases your empty lines are not separating logical sections and consequently they shouldn't be there. When there is a lot of code after the return I could agree that are different logical sections and that this improve readability, but when there is only one more line I really don't see the point.